pom.xml

```diff
@@ -462,14 +462,14 @@
 				<version>1.039</version>
 			</dependency>
 			<dependency>
-				<groupId>javax.xml.bind</groupId>
-				<artifactId>jaxb-api</artifactId>
-				<version>2.3.1</version>
+				<groupId>jakarta.xml.bind</groupId>
+				<artifactId>jakarta.xml.bind-api</artifactId>
+				<version>4.0.0</version>
```

Comment:
I think versions should be declared in this top pom.xml only (note this is the `dependencyManagement` section) and not in the modules. Then we have centralised version management.

Reply:
Removed explicit versions and scope from POM as required.